packages/order-watcher/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../../tsconfig",
     "compilerOptions": {
+        "downlevelIteration": true,
```

Comment:
Let's do more research into this and if there are no large downsides, let's move to top-level `tsconfig.json`.

Reply:
https://blog.mariusschulz.com/2017/06/30/typescript-2-3-downlevel-iteration-for-es3-es5

TL;DR
The only downside is that if we run in the environment with no [Interator support](https://caniuse.com/#search=iterator) we need to include the polyfill for it.